wordpress-mobile/gutenberg-mobile
# GB PR: Add border colour tokens

Updates the colours for the border. 

To test:
See https://github.com/WordPress/gutenberg/pull/25429 

PR submission checklist:
